### PR TITLE
WooGraphQL session documentation added.

### PR DESCRIPTION
**Pull request by kidunot89**
### Your checklist for this pull request
Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.

🚨Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository.

- [x] Make sure you are making a pull request against the **develop branch** (left side). Also you should start *your branch* off *our develop*.
- [x] Make sure you are requesting to pull request from a **topic/feature/bugfix branch** (right side). Don't pull request from your master!

What does this implement/fix? Explain your changes.
---------------------------------------------------
Adds in depth documentation for WooGraphQL sessions.

Does this close any currently open issues?
------------------------------------------
…

Any relevant logs, error output, GraphiQL screenshots, etc?
-------------------------------------
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

Any other comments?
-------------------
…

Where has this been tested?
---------------------------
**Operating System:** Ubuntu 20.04

**WordPress Version:** 5.6

### CODE REVIEW

File: docs/content/session/1-index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: "Why is this important?"
```
Comment (jacobarriola) [inline docs/content/session/1-index.md:2]:
```suggestion
title: "WooGraphQL session management"
```

File: docs/content/session/1-index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+metaTitle: "Why is understanding a WooGraphQL's session management important? | WooGraphQL Docs | AxisTaylor"
+metaDescription: "Learn the responsibilites and capabilities of the custom session handler WooGraphQL uses."
+---
+
+Typically GraphQL requests don't have an effect on the user context of the environment when a request is sent. This is due to user context or custom context rarely ever being necessary when querying for public data. However when dealing with private or personal data, your gonna wanna hide values, fields, or possibly types behind user context.
```
[the remark this answers is not in the document]
Comment (jacobarriola) [reply docs/content/session/1-index.md:7]: 'of the environment' really doesn't make much sense to me though.

[the remark this answers is not in the document]
Comment (kidunot89) [reply docs/content/session/1-index.md:7]: It should probably be **PHP environment**

File: docs/content/session/1-index.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+When I refer to user context, I'm referring to changes to the environment that cause user checks like the following
```
[the remark this answers is not in the document]
Comment (kidunot89) [reply docs/content/session/1-index.md:9]: This point is actually no good. You're comment in the suggestion above :point_up: says more about this, seeing as this is supposed explain what **user context of the environment** is :sweat_smile:.

I'll rewrite this one.

File: docs/content/session/1-index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kidunot89) [reply docs/content/session/1-index.md:7]: @jacobarriola :point_up: I edited your suggestion.

**Comment by TheAggressive** [comment]
Anyone have a working solution? Trying to implement a cart but having issues without an example and the docs are "coming soon"

**Comment by kidunot89** [comment]
@TheAggressive Sorry about the delay. I have some more documentation to add, and need to update what here. I'll be trying to get this merge in early to mid January.

**Comment by TheAggressive** [comment]
@kidunot89 appreciate all your effort!

**Pull request closed without merging**